test-worker/test/asset-compute/worker/watermark-percent-on-jpg/params.json

```
@@ -0,0 +1,11 @@
+{
+	"fmt": "jpg",
+    "watermark": {
+        "name": "watermark",
+        "placement": "Center",
+        "watermarkContent": "watermark.png",
+        "widthPercent": 25
```

```suggestion
        "widthPercent": 25.5
```
(To test float value handling).

---

I would also add a negative test, where widthPercentage > 100, to validate error handling.